packages/common/src/permissions/_internal/checkAvailability.ts

```diff
@@ -36,13 +36,23 @@ export function checkAvailability(
 
     // reduce over the policy.availability array, adding a check for each
     // value that is not included in the contextAvailability array
+    // the 'flag' is only used when we want to gate the feature behind
+    // a feature flag, e.g. ?pe=hub:card:follow
     checks = policy.availability.reduce((acc: IPolicyCheck[], value) => {
       let result: PolicyResponse = "granted";
       if (!contextAvailability.includes(value)) {
-        result = `not-${value}-org` as PolicyResponse;
+        if (value === "flag") {
```

Comment:
I may be totally missing something here, but how does "flag" ever get added to the `contextAvailability` array? It looks like the other options get conditionally added to `contextAvailability` above based on contextual information.

Reply:
ah, I see - thank you for the explanation!